include/z64cutscene_commands.h

```diff
@@ -269,7 +269,7 @@
 /**
  * Marks the end of a cutscene script.
  */
-#define CS_END() 0xFFFFFFFF, 0x00000000
+#define CS_END_SCRIPT() 0xFFFFFFFF, 0x00000000
```

Comment:
You should be able to use `CS_CMD_END_OF_SCRIPT` where it has all the Fs.
Additionally, I think they should match (both have "of", or dont). 

I lean toward having it, so
`#define CS_END_OF_SCRIPT() CS_CMD_END_OF_SCRIPT, 0x00000000`

Reply:
Also the comment above the definition can be more clear:
"Marks the end of a command list in a cutscene script. Processing for the current frame will finish."
or something along those lines

Reply:
probably `#define CS_END_OF_SCRIPT() CS_CMD_END_OF_SCRIPT, CMD_W(0)` to match the other commands